app/assets/javascripts/practice_editor_overview.es6

```diff
@@ -77,17 +82,96 @@
         });
     }
 
+    function addDisableAttrAndColor(labelSelector, inputSelector) {
+        const disabledColor = '#c9c9c9';
+        labelSelector.css('color', disabledColor);
```

Comment:
Could we make a class to apply to the inputSelector when it is disabled and apply that class so we don't have to manage the css in this file?